### DISCUSSION TITLE
Old project with index.min.js

### DISCUSSION THREAD

**Discussion by codedbyfrancis** [mdbootstrap/TW-Elements#1721] [Chat about dev stuff]
Hello,
I have an old project using tw-elements 1.0.0-alpha12 and Nuxt 2. Everything was working perfectly but I abandoned it last year. Now when I run the project again my accordion and modals don't work anymore, when I click on accordion it does nothing the same with modal. I'm not sure what is the cause but I was using CDN from https://cdn.jsdelivr.net/npm/tw-elements/dist/js/index.min.js but now it is not found. I tried using https://cdn.jsdelivr.net/npm/tw-elements/dist/js/tw-elements.umd.min.js but still not working. Where can I get the old one?
Thanks.

**Comment by EliaGut** [comment]
Are you a vegan? If not, the no link for you, to torture animals is BAD. But I have the old version link.

**Comment by juujisai** [comment]
Hi @codedbyfrancis 
You can find the alpha13 cdn here:

- https://cdn.jsdelivr.net/npm/tw-elements@1.0.0-alpha13/dist/js/index.min.js
- https://cdn.jsdelivr.net/npm/tw-elements@1.0.0-alpha13/dist/css/index.min.css
